### DISCUSSION TITLE
Welcome to the Counterparty Discussion Forum!

### DISCUSSION THREAD

**Discussion by loon3** [CounterpartyXCP/CIPs#85] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by rigeback** [comment]
I'm helping a friend of mine who was paid with Troptions and I am looking for information regarding payment for crypto or a way to use for purchasing goods.
I came across this team https://github.com/troptions/troptions and it got me thinking the answer maybe here on GitHub!
In so far I have not been able to find a wallet or broker that accepts these keys?
Any info would be much appreciated
